src/winuwp/runner_uwp/flutter_frameworkview.cpp

```diff
@@ -0,0 +1,155 @@
+        OutputDebugString(L"Couldn't disable layout scaling");
+      }
+    }
+
+    main_view_ = applicationView;
+    main_view_.Activated({this, &FlutterFrameworkView::OnActivated});
+  }
+
+  // |winrt::Windows::ApplicationModel::Core::IFrameworkView|
+  void Uninitialize() {
+    main_view_.Activated(nullptr);
+    main_view_ = nullptr;
+  }
+
+  // |winrt::Windows::ApplicationModel::Core::IFrameworkView|
+  void Load(winrt::hstring const &) {}
+
+  // |winrt::Windows::ApplicationModel::Core::IFrameworkView|
+  void Run() {
+    winrt::Windows::UI::Core::CoreWindow window =
+        winrt::Windows::UI::Core::CoreWindow::GetForCurrentThread();
+
+    winrt::Windows::UI::Core::CoreDispatcher dispatcher = window.Dispatcher();
+    dispatcher.ProcessEvents(
+        winrt::Windows::UI::Core::CoreProcessEventsOption::ProcessUntilQuit);
+  }
+
+  // |winrt::Windows::ApplicationModel::Core::IFrameworkView|
+  winrt::Windows::Foundation::IAsyncAction
+  SetWindow(winrt::Windows::UI::Core::CoreWindow const &window) {
+
+    // Capture reference to window.
+    window_ = window;
+
+    // Lay out the window's content within the region occupied by the
+    // CoreWindow.
+    auto appView = winrt::Windows::UI::ViewManagement::ApplicationView::
+        GetForCurrentView();
+
+    appView.SetDesiredBoundsMode(winrt::Windows::UI::ViewManagement::
+                                     ApplicationViewBoundsMode::UseCoreWindow);
+
+    // Configure folder paths.
+    try {
+      winrt::Windows::Storage::StorageFolder folder =
+          winrt::Windows::ApplicationModel::Package::Current()
+              .InstalledLocation();
+
+      winrt::Windows::Storage::StorageFolder assets =
+          co_await folder.GetFolderAsync(L"Assets");
+      winrt::Windows::Storage::StorageFolder data =
+          co_await assets.GetFolderAsync(L"data");
+      winrt::Windows::Storage::StorageFolder flutter_assets =
+          co_await data.GetFolderAsync(L"flutter_assets");
+      winrt::Windows::Storage::StorageFile icu_data =
+          co_await data.GetFileAsync(L"icudtl.dat");
+
+#if NDEBUG
+      winrt::Windows::Storage::StorageFile aot_data =
+          co_await data.GetFileAsync(L"app.so");
+#endif
+
+      std::wstring flutter_assets_path{flutter_assets.Path()};
+      std::wstring icu_data_path{icu_data.Path()};
+      std::wstring aot_data_path {
+#if NDEBUG
+        aot_data.Path()
+#endif
+      };
+
+      flutter::DartProject project(flutter_assets_path, icu_data_path,
+                                   aot_data_path);
+
+      // Construct viewcontroller using the Window and project
+            flutter_view_controller_ = std::make_unique<flutter::FlutterViewController>(
+                static_cast<ABI::Windows::ApplicationModel::Core::CoreApplicationView*>(winrt::get_abi(main_view_)),
+                static_cast<ABI::Windows::ApplicationModel::Activation::IActivatedEventArgs*>(winrt::get_abi(launch_args_)),
+                project);
```

Comment:
The formatting for lines 112-116 looks wrong.

Reply:
fixed in next commit